src/js/ApiGetter/ApiGetter.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import '../../scss/main.scss';
```

Comment:
Leave a spare line between library imports, project's components and styles, please